Validation/Configuration/python/globalValidation_cff.py

```diff
@@ -181,6 +188,13 @@
     + calotowersValidationSequence
 )
 
+globalValidationHCALOnly = cms.Sequence(
+      hcalSimHitsValidationSequence
+    + hcaldigisValidationSequence
+    + hcalSimHitStudy
+    + hcalRecHitsOnlyValidationSequence
+)
+
 globalValidationHGCal = cms.Sequence(hgcalValidation)
```

Comment:
@jfernan2  I'm noticing that in 11_1 there is not a ```globalPrevalidationHGCal``` , while in master it is there:
https://github.com/cms-sw/cmssw/blob/240308ae793579cf2b589485497289ef153be2fc/Validation/Configuration/python/globalValidation_cff.py#L199

Reply:
That was added in #29844, unrelated to this PR